src/chrome/content/rules/Beatport.com.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	Non-functional hosts:
+		Redirect to HTTP:
+		- sounds.beatport.com
+-->
+<ruleset name="Beatport.com">
+
+	<target host="beatport.com" />
+	<target host="www.beatport.com" />
+	<target host="accounts.beatport.com" />
+	<target host="baseware.beatport.com" />
+	<target host="dj.beatport.com" />
+	<target host="embed.beatport.com" />
+	<target host="geo-media.beatport.com" />
```

Comment:
Can you add a test url such as http://geo-media.beatport.com/image/2d4b4b06-d0a6-4d8d-b7d1-b68ed3bb6708.jpg please?